tests/README.md

```diff
@@ -0,0 +1,28 @@
+# blaseball-mike tests
+
+### Run test suite
+```shell
+# Install dependancies
+pip install pytest pytest-cov pytest-recording
+
+# Run whole test suite
+pytest
+```
+---
+### Run a single test
+```shell
+# If you know the exact test (This tests player modifications)
+pytest tests/test_player.py::TestPlayer::test_modifications
+
+# If you want to test by keywords (This tests game bet payouts)
+pytest -k "TestGame and payout"
+```
+---
+### I'm getting 'vcr.errors.CannotOverwriteExistingCassetteException' errors...
+We use [vcrpy](https://vcrpy.readthedocs.io/en/latest/index.html) to record network calls
```

Comment:
update link?

Reply:
That's the generic `vcr` documentation, but I can add a link to the `pytest-recording` github. Their documentation is a lot worse though.